helm/squidex/values.yaml

```diff
@@ -32,29 +32,71 @@ ingress:
   #      - chart-example.local
 
 env:
-  URLS__BASEURL: null
```

Comment:
If we have so many settings we should sort them alphabetically.

Reply:
sorted the env settings. Added more values for the helm show values squidex/squidex cmd. You can export these values which is convenient for deploying the app.